From bb765360408b53aeb5042fcbefa4ae5c448288f6 Mon Sep 17 00:00:00 2001
From: Ying Wang <wangying@google.com>
Date: Tue, 16 Dec 2014 12:33:47 -0800
Subject: [PATCH 05/11] msm8974: Cleanup obsolete LOCAL_PRELINK_MODULE
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Bug: 18675947
Change-Id: I2f4495b89633c9e72e229b9279b1c5f6cdead740
See I6a8f5f5317d1865f0a5cf7ce70d74d3f70671987
Also added the libloc_api-rpc-{glue,stub} MKs
Signed-off-by: D. Andrei Măceș <Andrei.Maces@alumni.nd.edu>
---
 msm8974/core/Android.mk                                        | 2 --
 .../libloc_api-rpc-50001/libloc_api-rpc-glue/Android.mk        | 2 --
 .../libloc_api-rpc-50001/libloc_api-rpc-stub/Android.mk        | 1 -
 msm8974/loc_api/libloc_api_50001/Android.mk                    | 3 ---
 msm8974/utils/Android.mk                                       | 2 --
 5 files changed, 10 deletions(-)

diff --git a/msm8974/core/Android.mk b/msm8974/core/Android.mk
index c7ea75c..8ca6560 100644
--- a/msm8974/core/Android.mk
+++ b/msm8974/core/Android.mk
@@ -47,8 +47,6 @@ LOCAL_COPY_HEADERS:= \
     loc_core_log.h \
     LocAdapterProxyBase.h
 
-LOCAL_PRELINK_MODULE := false
-
 include $(BUILD_SHARED_LIBRARY)
 
 endif # not BUILD_TINY_ANDROID
diff --git a/msm8974/loc_api/libloc_api-rpc-50001/libloc_api-rpc-glue/Android.mk b/msm8974/loc_api/libloc_api-rpc-50001/libloc_api-rpc-glue/Android.mk
index bb3d924..3d09024 100644
--- a/msm8974/loc_api/libloc_api-rpc-50001/libloc_api-rpc-glue/Android.mk
+++ b/msm8974/loc_api/libloc_api-rpc-50001/libloc_api-rpc-glue/Android.mk
@@ -31,8 +31,6 @@ LOCAL_SHARED_LIBRARIES:= \
 LOCAL_STATIC_LIBRARIES := \
     libloc_api_rpcgen
 
-LOCAL_PRELINK_MODULE:= false
-
 LOCAL_C_INCLUDES:= \
     $(LOCAL_PATH) \
     $(LOCAL_PATH)/rpc_inc \
diff --git a/msm8974/loc_api/libloc_api-rpc-50001/libloc_api-rpc-stub/Android.mk b/msm8974/loc_api/libloc_api-rpc-50001/libloc_api-rpc-stub/Android.mk
index c0987e6..1f542e4 100644
--- a/msm8974/loc_api/libloc_api-rpc-50001/libloc_api-rpc-stub/Android.mk
+++ b/msm8974/loc_api/libloc_api-rpc-50001/libloc_api-rpc-stub/Android.mk
@@ -32,5 +32,4 @@ LOCAL_COPY_HEADERS += inc/loc_api_rpcgen_cb_rpc.h
 LOCAL_COPY_HEADERS += inc/loc_apicb_appinit.h
 
 LOCAL_LDLIBS += -lpthread
-LOCAL_PRELINK_MODULE := false
 include $(BUILD_STATIC_LIBRARY)
diff --git a/msm8974/loc_api/libloc_api_50001/Android.mk b/msm8974/loc_api/libloc_api_50001/Android.mk
index e38befa..e75db11 100644
--- a/msm8974/loc_api/libloc_api_50001/Android.mk
+++ b/msm8974/loc_api/libloc_api_50001/Android.mk
@@ -54,8 +54,6 @@ LOCAL_COPY_HEADERS:= \
    loc_eng_msg.h \
    loc_eng_log.h
 
-LOCAL_PRELINK_MODULE := false
-
 include $(BUILD_SHARED_LIBRARY)
 
 include $(CLEAR_VARS)
@@ -97,7 +95,6 @@ ifneq ($(filter msm8084,$(TARGET_BOARD_PLATFORM)),)
   LOCAL_CFLAGS += -DPLATFORM_MSM8084
 endif
 
-LOCAL_PRELINK_MODULE := false
 LOCAL_MODULE_RELATIVE_PATH := hw
 
 include $(BUILD_SHARED_LIBRARY)
diff --git a/msm8974/utils/Android.mk b/msm8974/utils/Android.mk
index f34e101..955e20d 100644
--- a/msm8974/utils/Android.mk
+++ b/msm8974/utils/Android.mk
@@ -54,8 +54,6 @@ LOCAL_MODULE := libgps.utils
 
 LOCAL_MODULE_TAGS := optional
 
-LOCAL_PRELINK_MODULE := false
-
 include $(BUILD_SHARED_LIBRARY)
 endif # not BUILD_TINY_ANDROID
 endif # BOARD_VENDOR_QCOM_GPS_LOC_API_HARDWARE
-- 
2.17.1

